pkg/semtech-loramac/doc.txt

```diff
@@ -95,12 +95,6 @@
  *         return 1;
  *     }
  *
- *     /* 5. wait for any potentially received data */
```

Comment:
Document starting and usage of a receiving thread.

Reply:
Done